tests/Tests/test-minimal-image.robot

```diff
@@ -0,0 +1,114 @@
+*** Settings ***
+Resource        ../Resources/ODS.robot
+Resource        ../Resources/Common.robot
+Library         DebugLibrary
+Library         JupyterLibrary
+Suite Setup      Begin Web Test
+Suite Teardown   End Web Test
+
+*** Variables ***
+
+
+*** Test Cases ***
+Open ODH Dashboard
+  [Tags]  Sanity
+  #Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+  Login To ODH Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+  Wait For Condition  return document.title == "Red Hat OpenShift Data Science Dashboard"
+
+Can Launch Jupyterhub
+  [Tags]  Sanity
+  Launch JupyterHub From ODH Dashboard Dropdown
+
+Can Login to Jupyterhub
+  [Tags]  Sanity
+  Login To Jupyterhub  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+  ${authorization_required} =  Is Service Account Authorization Required
+  Run Keyword If  ${authorization_required}  Authorize jupyterhub service account
+
+Can Spawn Notebook
+  [Tags]  Sanity
+  # We need to skip this testcase if the user has an existing pod
+  ${spawner_visible} =  JupyterHub Spawner Is Visible
+  Skip If  ${spawner_visible}!=True  The user has an existing notebook pod running
+  Select Notebook Image  s2i-minimal-notebook
+  #This env is required until JupyterLab is the default interface in RHODS
+  Add Spawner Environment Variable  JUPYTER_ENABLE_LAB  true
+  Spawn Notebook
+
+Can Launch Python3 Smoke Test Notebook
+  [Tags]  Sanity
+
+  Wait for JupyterLab Splash Screen
+
+
+  # Sometimes we get a modal pop-up upon server spawn asking to select a kernel for the notebooks
+  ${is_kernel_selected} =  Run Keyword And Return Status  Page Should Not Contain Element  xpath=//div[@class="jp-Dialog-buttonLabel"][.="Select"]
+  Run Keyword If  not ${is_kernel_selected}  Click Element  xpath=//div[@class="jp-Dialog-buttonLabel"][.="Select"]
+
+  ${is_launcher_selected} =  Run Keyword And Return Status  JupyterLab Launcher Tab Is Selected
+  Run Keyword If  not ${is_launcher_selected}  Open JupyterLab Launcher
+  Launch a new JupyterLab Document
+
+  Close Other JupyterLab Tabs
+
+  ##################################################
+  # Manual Notebook Input
+  ##################################################
+  # Sometimes the kernel is not ready if we run the cell too fast
+  Sleep  5
+  Run Cell And Check For Errors  !pip install boto3
+
+  Add and Run JupyterLab Code Cell  import os
+  Run Cell And Check Output  print("Hello World!")  Hello World!
+
+  #Needs to change for RHODS release
+  Run Cell And Check Output  !python --version  Python 3.8.3
+  #Run Cell And Check Output  !python --version  Python 3.8.7
+
+  Capture Page Screenshot
+  JupyterLab Code Cell Error Output Should Not Be Visible
+
+  ##################################################
+  # Git clone repo and run existing notebook
+  ##################################################
+  #Maybe Open JupyterLab Sidebar  File Browser
```

Comment:
we should delete ununsed code along with its comments

Reply:
Done in 0e88e30